drivers/soundwire/amd_init.c

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: GPL-2.0+
```

Comment:
please check with your lawyers. The code is directly inspired from Intel and released with a GPL-2.0-only license.

// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)

There is no '2.0 or later' in the initial Intel code.

Reply:
we will check